master/buildbot/test/regressions/test_steps_shell_WarningCountingShellCommand.py

```diff
@@ -48,3 +49,10 @@ def testSuppressingLinelessWarningsPossible(self):
         # list of warnings
         expectedWarnings = 0
         self.assertEquals(len(warnings), expectedWarnings)
+
+    # To test if we can catch the exception when and return an error
+    # the suppressionFile does not exist or no right to access the file.
+    def testWithNotExistSuppressionFile(self):
+        # Use a filename which doesn't exist
+        w = WarningCountingShellCommand(
+            suppressionFile='impossiblefilename.txt')
```

Comment:
you can look in other tests for self.expectCommands there are plenty of example tests to simulate remote commands